CMakeLists.txt

```
@@ -24,6 +31,19 @@ if(POLICY CMP0077)
   cmake_policy(SET CMP0077 NEW) #option does nothing when a normal variable of the same name exists.
 endif()
 
+function(target_add_awslc_include_paths target_name scope)
+  if(ARGC EQUAL 2 OR ARGC GREATER 2)
```

Should be?
```suggestion
  if(ARGC GREATER 2)
```
since `ARGV2` is index 2, in the 0-start index argument list.